ci: error: one or more PCH files were found, but they were invalid

This has been reported by multiple contributors. I assume some sort of caching issue? Seems to happen intermittently. https://cirrus-ci.com/task/5028170181640192?logs=ci#L2664:
```bash
make[4]: Entering directory '/ci_container_base/depends/work/build/i686-pc-linux-gnu/qt/5.15.5-ba80ab1130c/qtbase/src/testlib'
make[5]: Entering directory '/ci_container_base/depends/work/build/i686-pc-linux-gnu/qt/5.15.5-ba80ab1130c/qtbase/src/plugins/platforminputcontexts'
make[5]: Entering directory '/ci_container_base/depends/work/build/i686-pc-linux-gnu/qt/5.15.5-ba80ab1130c/qtbase/src/plugins/platforms'
cc1plus: error: one or more PCH files were found, but they were invalid
cc1plus: error: use -Winvalid-pch for more information
cc1plus: error: one or more PCH files were found, but they were invalid
cc1plus: error: use -Winvalid-pch for more information
cc1plus: fatal error: .pch/Qt5Test: No such file or directory
cc1plus: fatal error: .pch/Qt5Test: No such file or directory
compilation terminated.
compilation terminated.
cc1plus: error: one or more PCH files were found, but they were invalid
cc1plus: error: use -Winvalid-pch for more information
cc1plus: fatal error: .pch/Qt5Test: No such file or directory
compilation terminated.
cc1plus: error: one or more PCH files were found, but they were invalid
cc1plus: error: use -Winvalid-pch for more information
make[4]: *** [Makefile:2113: .obj/qtestcase.o] Error 1
make[4]: *** Waiting for unfinished jobs....
make[4]: *** [Makefile:2292: .obj/qtestlog.o] Error 1
make[4]: *** [Makefile:2435: .obj/qtesttable.o] Error 1
cc1plus: fatal error: .pch/Qt5Test: No such file or directory
compilation terminated.
make[5]: Entering directory '/ci_container_base/depends/work/build/i686-pc-linux-gnu/qt/5.15.5-ba80ab1130c/qtbase/src/plugins/platformthemes'
make[4]: *** [Makefile:2578: .obj/qtestdata.o] Error 1
cc1plus: error: one or more PCH files were found, but they were invalid
cc1plus: error: use -Winvalid-pch for more information
cc1plus: fatal error: .pch/Qt5Test: No such file or directory
compilation terminated.
make[4]: *** [Makefile:2944: .obj/qtestresult.o] Error 1
make[4]: Leaving directory '/ci_container_base/depends/work/build/i686-pc-linux-gnu/qt/5.15.5-ba80ab1130c/qtbase/src/testlib'
make[3]: *** [Makefile:298: sub-testlib-make_first] Error 2
make[3]: *** Waiting for unfinished jobs....
```

Not sure why Qt is even trying to use pch on a Linux build, as it was meant to be disabled by #20520.

--- a/ci/test/04_install.sh
+++ b/ci/test/04_install.sh
@@ -21,6 +21,8 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
       "${BASE_READ_ONLY_DIR}"
   docker volume create "${CONTAINER_NAME}_ccache" || true
   docker volume create "${CONTAINER_NAME}_depends" || true
+  docker volume create "${CONTAINER_NAME}_depends_sources" || true
+  docker volume create "${CONTAINER_NAME}_depends_SDKs_android" || true
   docker volume create "${CONTAINER_NAME}_previous_releases" || true
 
   if [ -n "${RESTART_CI_DOCKER_BEFORE_RUN}" ] ; then
@@ -36,7 +38,9 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
   CI_CONTAINER_ID=$(docker run --cap-add LINUX_IMMUTABLE $CI_CONTAINER_CAP --rm --interactive --detach --tty \
                   --mount "type=bind,src=$BASE_READ_ONLY_DIR,dst=$BASE_READ_ONLY_DIR,readonly" \
                   --mount "type=volume,src=${CONTAINER_NAME}_ccache,dst=$CCACHE_DIR" \
-                  --mount "type=volume,src=${CONTAINER_NAME}_depends,dst=$DEPENDS_DIR" \
+                  --mount "type=volume,src=${CONTAINER_NAME}_depends,dst=$DEPENDS_DIR/built" \
+                  --mount "type=volume,src=${CONTAINER_NAME}_depends_sources,dst=$DEPENDS_DIR/sources" \
+                  --mount "type=volume,src=${CONTAINER_NAME}_depends_SDKs_android,dst=$DEPENDS_DIR/SDKs/android" \
                   --mount "type=volume,src=${CONTAINER_NAME}_previous_releases,dst=$PREVIOUS_RELEASES_DIR" \
                   --env-file /tmp/env \
                   --name "$CONTAINER_NAME" \

